usedapp-rootstock/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "usedapp-rootstock",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@usedapp/core": "^1.2.3-dev.f3c2181",
```

Comment:
when there is a versioned release, let's remember to loop back here and update this